app/upgrades/v6/upgrades.go

```diff
@@ -0,0 +1,83 @@
+package v6
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	bankkeeper "github.com/cosmos/cosmos-sdk/x/bank/keeper"
+	paramskeeper "github.com/cosmos/cosmos-sdk/x/params/keeper"
+	slashingkeeper "github.com/cosmos/cosmos-sdk/x/slashing/keeper"
+	stakingkeeper "github.com/cosmos/cosmos-sdk/x/staking/keeper"
+	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
+
+	ibctransferkeeper "github.com/cosmos/ibc-go/v3/modules/apps/transfer/keeper"
+
+	evmtypes "github.com/evmos/ethermint/x/evm/types"
+	feemarkettypes "github.com/evmos/ethermint/x/feemarket/types"
+
+	v5 "github.com/evmos/evmos/v6/app/upgrades/v5"
+	"github.com/evmos/evmos/v6/types"
+	claimskeeper "github.com/evmos/evmos/v6/x/claims/keeper"
+)
+
+// CreateUpgradeHandler creates an SDK upgrade handler for v5
+func CreateUpgradeHandler(
+	mm *module.Manager,
+	configurator module.Configurator,
+	bk bankkeeper.Keeper,
+	ck *claimskeeper.Keeper,
+	sk stakingkeeper.Keeper,
+	pk paramskeeper.Keeper,
+	tk ibctransferkeeper.Keeper,
+	xk slashingkeeper.Keeper,
+) upgradetypes.UpgradeHandler {
+	return func(ctx sdk.Context, _ upgradetypes.Plan, vm module.VersionMap) (module.VersionMap, error) {
+		logger := ctx.Logger().With("upgrade", UpgradeName)
+
+		// modify fee market parameter defaults through global
+		feemarkettypes.DefaultMinGasPrice = v5.MainnetMinGasPrices
```

Comment:
@fedekunze We might want to start with a lower min gas price and then raise it though governance to make sure, that the gas prices don't skyrocket by accident.

Reply:
isn't it effectively the same outcome as setting it here?